src/test/java/net/minecraftforge/test/TestCapabilityMod.java

```diff
@@ -160,5 +202,10 @@ public void readNBT(Capability<IExampleCapability> capability, IExampleCapabilit
         public String getOwnerType(){
             return "Default Implementation!";
         }
+        public boolean getVal() {
```

Comment:
Eh... If you think this is ok, why don't you approve this pull and assign Lex Manos? I would like to see this bugfix added.